src/bun.js/webcore/response.zig

```diff
@@ -1280,6 +1280,73 @@ pub const Fetch = struct {
                     error.FailedToOpenSocket => bun.String.static("Was there a typo in the url or port?"),
                     error.TooManyRedirects => bun.String.static("The response redirected too many times. For more information, pass `verbose: true` in the second argument to fetch()"),
                     error.ConnectionRefused => bun.String.static("Unable to connect. Is the computer able to access the url?"),
+
+                    error.UNABLE_TO_GET_ISSUER_CERT => bun.String.static("unable to get issuer certificate"),
```

Comment:
I have a feeling this code and other parts of this file would be a lot simpler if `us_bun_verify_error_t` had some more methods defined on it.